packages/ruleset/src/functions/api-symmetry.js

```diff
@@ -253,15 +268,21 @@ function checkForGraphFragmentPattern(
     ['allOf', 'oneOf', 'anyOf'].forEach(applicator => {
       if (
         Array.isArray(variant[applicator]) &&
-        variant[applicator].length > 0 &&
-        !variant[applicator].reduce(
-          (previousResult, v) =>
-            previousResult &&
-            isGraphFragment(v, canonical, canonicalPath, true),
-          true
-        )
+        variant[applicator].length > 0
       ) {
-        result = false;
+        if (
+          !variant[applicator].reduce(
+            (previousResult, v) =>
+              previousResult &&
+              isGraphFragment(v, canonical, canonicalPath, true),
+            true
+          )
+        ) {
+          logger.info(
+            `${ruleId}: variant schema applicator '${applicator}' is not a graph fragment of the canonical schema`
+          );
+          result = false;
+        }
```

Comment:
I'm not sure I understand this change - isn't it logically equivalent but with a log statement added? 🙂 

Let me know if I'm missing something